liquidator/.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
```

Comment:
Can you explain for a second why we want a separate eslint config? would it not be better to apply this one later down (with the rest of the linter logic in the main `package.json`) or is it better practice to have this kind of linter on a _per package_ basis? I'm not sure what the standard is for mono-repos & the linters.

Reply:
I'm actually not too sure whats the best practice either. There's arguments [for](https://verdaccio.org/blog/2019/09/07/managing-multiples-projects-with-lerna-and-yarn-workspaces#managing-dependencies-and-devdependencies) and [against](https://eslint.org/docs/developer-guide/shareable-configs) having separated eslint configs.

The only reason why I made a dedicated eslint config was because we were including `jsx` configurations for the liquiation bot.

What are your thoughts on this?

Reply:
I see, and I'm also not sure what's best practice. For me I think this comes down to how we expect the packages to be used: if we expect people to install individual packages on their own then I'd say they should have their own linter config. For example if someone installs UMA.js or some npm SDK then I think we would want them to have their own linter config. However, out of that context when the repo is used as a collective set of packages I think it's better to have one linter.

I think for now it's best to keep this as one linter for all packages that all use the same husky hook and the whole repo follows the same pattern. 

@nicholaspai, @mrice32 & @daywiss what do you guys think?

Reply:
@chrismaree while I think the intent is for people to install individual packages, I think its easiest to reason about one repo-wide linting file. If anything, I don't want to group in the linting config with the rest of this already large PR.